spec/requests/v1/risk_areas/create_risk_areas_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Create Risk Area', type: :request do
+  describe 'POST /v1/risk_areas' do
+    context 'when risk area is created' do
+      it 'returns status 201 (:created)' do
+        post '/v1/risk_areas', :params => { "risk_area": { "type": "flooding", "description": "Inundou" } }
```

Comment:
```suggestion
        post '/v1/risk_areas', params: { 
          risk_area: { 
            type: 'flooding', 
            description: 'Inundou' 
          } 
        }
```
You can change these params into a symbol, if you want to. :slightly_smiling_face: